packages/rum-core/src/domain/rumEventsCollection/action/trackClickActions.ts

```diff
@@ -167,17 +166,50 @@ export function trackClickActions(
   }
 }
 
-function listenClickEvents(callback: (clickEvent: MouseEvent & { target: Element }) => void) {
-  return addEventListener(
+function listenClickEvents(
```

Comment:
[DRAFT] because this function will grow over time, I'll extract it to another file when finalizing the PR